sdk/keyvault/azure-security-keyvault-keys/README.md

```diff
@@ -281,7 +281,7 @@ std::cout << updatedKey.Properties.UpdatedOn->ToString();
 
 `StartDeleteKey` starts a long-running operation to delete a key previously stored in the Azure Key Vault.
 You can retrieve the key immediately without waiting for the operation to complete.
-When [soft-delete][soft_delete] is not enabled for the Azure Key Vault, this operation permanently deletes the key.
+When [soft-delete](https://docs.microsoft.com/en-us/azure/key-vault/general/soft-delete-overview) is not enabled for the Azure Key Vault, this operation permanently deletes the key.
```

Comment:
Btw, the link verification is failing in CI because we are using an en-us specific link.
We can fix it by using the general one:
```suggestion
When [soft-delete](https://docs.microsoft.com/azure/key-vault/general/soft-delete-overview) is not enabled for the Azure Key Vault, this operation permanently deletes the key.
```